test/specs/scale.time.tests.js

```diff
@@ -1235,4 +1235,37 @@ describe('Time scale tests', function() {
       });
     });
   });
+
+  it('should pass default locale to date adapter', function() {
+    var chart = window.acquireChart({
+      type: 'line',
+      data: {},
+      options: {
+        scales: {
+          x: {
+            type: 'time'
+          },
+        }
+      }
+    });
+
+    expect(chart.scales.x._adapter.chartOptions.locale).toBe('en-US');
```

Comment:
Won't this test fail if someone runs these tests when they are not in a locale that is `en-US`?